content/news/playtest-20190209.md

```diff
@@ -0,0 +1,34 @@
+
+A [new playtest build](/download/) is now available with a collection of fixes and improvements over [playtest-20190106](/news/playtest-20190106/).
+
+The last playtest made some big changes to how targeting interacts with the Fog of War, which broke a lot more than we expected. Rather than reverting these changes, we decided to go all-in, and replaced even more of the targeting code - a job that we had originally planned for the release after next.
+
+These new changes should fix not just the regressions in the last playtest, but also several other long-standing gameplay bugs and inconsistencies with the Fog of War, engineers, and aircraft. They also bring two long-requested features:
+
+* Tanks and other turreted units automatically target nearby units while moving
+* Force-firing with long-range artillery style units targets the ground, allowing attacks to continue when vision is lost under the Fog of War.
+
+We are excited for these new features, which should go a long way towards improving combat fluidness and the ability to micromanage your forces.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190209-opportunityfire.gif" alt="Tanks targeting while moving">
+
+Tanks now automatically target enemy forces while moving.
+</div>
+
+The audio notification changes introduced in the last playtest have been refined and expanded by adding click feedback to the sidebar, reducing the volume of the "cash tick" sounds, and normalizing the volume changes between different music tracks. The build is rounded out with fixes for bugs and crashes that were reported in the first playtest &mdash; see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/7b4a50814a012c161e68378f7af533e6c2dfd5f7) and [playtest-20190106](/news/playtest-20190106/) news post for more detail.
```

Comment:
Maybe better to put this into two sentences?

"The audio notification changes introduced in the last playtest have been refined and expanded by adding a sound feedback when clicking on items in the sidebar.  We also reduced the volume of the "cash tick" sounds, and normalized the volume changes between different music tracks."

Reply:
nvm